src/kava.js

```diff
@@ -70,6 +71,7 @@ class Kava extends BasePlugin {
     super(name, player, config);
     this._rateHandler = new KavaRateHandler();
     this._model = new KavaModel();
+    this._serviceUrl = Utils.Http.protocol + '//' + this.config.serviceUrl.replace(/(^\w+:|^)\/\//, '') + SERVICE_PATH;
```

Comment:
Use template strings since its much more readable

Reply:
why force the protocol? we assume that a working path is given and nobody promises that SERVICE_PATH is required.
Config should allow getting a fully functional URL as the service url can be proxied by another service fr example in an onPrem env.
The default protocol can be left // so we get the one the page has